packages/wethegit-react-hooks/eslint.config.js

```diff
@@ -0,0 +1,85 @@
+import { fixupConfigRules, fixupPluginRules } from "@eslint/compat"
+import typescriptEslint from "@typescript-eslint/eslint-plugin"
+import reactRefresh from "eslint-plugin-react-refresh"
+import globals from "globals"
+import tsParser from "@typescript-eslint/parser"
+import path from "node:path"
+import { fileURLToPath } from "node:url"
+import js from "@eslint/js"
+import { FlatCompat } from "@eslint/eslintrc"
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all,
+})
+
+export default [
+  {
+    ignores: [
+      "**/node_modules/",
+      "**/dist/",
+      "**/*.\\{css, scss}",
+      "**/.eslintrc.*",
+      "**/vite.config.ts",
+      "**/docs/",
+      "**/dist",
+      "**/node_modules",
+      "**/.turbo",
+      "**/vite.config.ts",
+    ],
+  },
+  ...fixupConfigRules(
+    compat.extends(
+      "eslint:recommended",
+      "plugin:@typescript-eslint/recommended",
+      "plugin:react/recommended",
+      "plugin:react/jsx-runtime",
+      "plugin:react-hooks/recommended",
+      "prettier"
+    )
+  ),
+  {
+    plugins: {
+      "@typescript-eslint": fixupPluginRules(typescriptEslint),
+      "react-refresh": reactRefresh,
+    },
+
+    languageOptions: {
+      globals: {
+        ...globals.browser,
+      },
+
+      parser: tsParser,
+      ecmaVersion: "latest",
+      sourceType: "module",
+    },
+
+    settings: {
+      react: {
+        version: "17.2",
```

Comment:
Should this be v18 @marlonmarcello ?

Reply:
Also looks like there's this `detect` option we might be able to use

```
    "react": {
      "version": "detect"
    }
```

Reply:
That's fixed, thanks @andrewrubin !